io/dev/user.clj

```diff
@@ -0,0 +1,120 @@
+;; Copyright 2020 Evident Systems LLC
+
+;; Licensed under the Apache License, Version 2.0 (the "License");
+;; you may not use this file except in compliance with the License.
+;; You may obtain a copy of the License at
+
+;;     http://www.apache.org/licenses/LICENSE-2.0
+
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+(ns ^:no-doc user
+  (:require [clojure.repl :refer :all]
+            [clojure.pprint :refer [pprint]]
+            [clojure.spec.test.alpha :as stest]
+            [clojure.tools.namespace.repl :refer [refresh refresh-all set-refresh-dirs]]
+            [kaocha.repl :as kaocha]
+            [criterium.core :as criterium]
+            [clj-async-profiler.core :as profiler]))
+
+(set! *warn-on-reflection* true)
+
+(ns-unmap *ns* 'test)
+
+(defn test []
+  (refresh)
+  (kaocha/run :unit-clj {:config-file "test/tests.edn"}))
+
+(defn instrument
+  []
+  (stest/instrument))
+
+(defn unstrument
+  []
+  (stest/unstrument))
+
+(comment
```

Comment:
Might be worth leaving some of these comment functions as a workflow function we could call with a test later